### ISSUE TITLE
JavaScript heap out of memory

### ISSUE THREAD

**Issue by JimmyMow** [electron-userland/electron-json-storage#98]
When importing the library in the renderer process I get

```
FATAL ERROR: CALL_AND_RETRY_LAST Allocation failed - JavaScript heap out of memory
```

when building config for electron renderer process via webpack. Anyone else experience this issue?

**Comment by jviotti** [comment]
Interesting. How much RAM do you have in your system? Do you have the code somewhere I can try from my machine?

**Comment by JimmyMow** [comment]
Yes the code is here on branch `feature/autopay`: https://github.com/LN-Zap/zap-desktop/tree/feature/autopay

It requires a outside dependency that is specific to the application though so it might be hard to get it up and running. I will try and spin up a trivial application to reproduce

**Comment by johnnyshankman** [comment]
i have the same issue. it's a 50mb user library/data file for cache purposes. tried your library as a last ditch effort since writing directly to userData only works when it's under about 5mb.
